.eslintrc

```diff
@@ -1,5 +1,14 @@
 {
-  "parserOptions": {
-    "sourceType": "module"
+  "extends": ["hypothesis", "plugin:jsx-a11y/recommended"],
+  "rules": {
+    // Suppressed to make ESLint v6 migration easier.
+    "no-prototype-builtins": "off",
+
+    // Handled by Prettier.
+    "comma-dangle": "off",
```

Comment:
It doesn't look like these two rule customizations are needed in this repository using the current version of ESLint and the current eslint-config-hypothesis package.